tools/privacyidea-token-janitor

```diff
@@ -462,7 +470,7 @@ def find(last_auth, assigned, active, tokeninfo_key, tokeninfo_value,
                 for token in tokens:
                     print(u",".join([u"'{0!s}'".format(x) for x in token]))
             else:
-                users = export_user_data(tlist)
+                users = export_user_data(tlist, attributes)
```

Comment:
Could please also add the ``--attributes`` to the function ``export_token_data`` in line 469.
This way we can always add attributes, when we do ``--listuser``.